contrib/debian/changelog.in

```diff
@@ -1,3 +1,9 @@
+openzfs-linux (@VERSION@-1) unstable; urgency=low
+
+  * OpenZFS @VERSION@ is tagged.
+
+ -- Umer Saleem <usaleem@ixsystems.com>  Wed, 15 Nov 2023 15:00:00 +0500
```

Comment:
Yes, I think should be. Ideally this should be the date of the release, but since we are not doing that, and the other option is to insert the current time stamp (at the time configure runs), which is not correct either, I think setting to a prior date should be alright.